matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/unnotifiedMatchesTable.js

```diff
@@ -77,12 +69,16 @@ var PhenoTips = (function (PhenoTips) {
     _showMatches : function()
     {
         var score = this._checkScore('show-matches-score', 'show-matches-messages');
-        if (score == undefined) {
+        var phenScore = this._checkScore('show-matches-phen-score', 'show-matches-messages');
+        var genScore = this._checkScore('show-matches-gen-score', 'show-matches-messages');
+        if (score == undefined && phenScore == undefined && genScore == undefined) {
```

Comment:
after this check we know at least one of them is defined - but some may still be undefined. So I think we should set all the ones that are `undefined` to 0 after this check

Reply:
also replace `&&` with `||`